src/ex4/dist/clients/pokemonClient.js

```diff
@@ -0,0 +1,38 @@
+const URL = "https://pokeapi.co/api/v2/pokemon";
```

Comment:
pokemon client logic should exists only in the server

Reply:
i'm useing it to render the pokemon pic in the frontend, is there any tip for best-practice?

Reply:
return the image url from server when fetching the pokemon